src/universal/modules/teamDashboard/components/TeamTasksHeader/TeamTasksHeader.js

```diff
@@ -93,48 +86,18 @@ const TeamTasksHeader = (props) => {
 TeamTasksHeader.propTypes = {
   children: PropTypes.any,
   dispatch: PropTypes.func.isRequired,
-  styles: PropTypes.object,
+  history: PropTypes.object,
   team: PropTypes.object.isRequired,
   teamMemberFilterId: PropTypes.string,
   teamMemberFilterName: PropTypes.string
 };
 
-const styleThunk = () => ({
-  button: {
-    ...inlineBlock,
-    color: appTheme.palette.mid,
-
-    ':hover': {
-      color: appTheme.palette.dark
-    },
-    ':focus': {
-      color: appTheme.palette.dark
-    }
-  },
-
-  filterRow: {
-    display: 'flex',
-    justifyContent: 'flex-end'
-  },
-
-  link: {
-    ...ib,
-    color: appTheme.palette.mid,
-
-    ':hover': {
-      color: appTheme.palette.dark
-    },
-    ':focus': {
-      color: appTheme.palette.dark
-    }
-  }
-});
-
 export default createFragmentContainer(
-  withStyles(styleThunk)(TeamTasksHeader),
+  withRouter(TeamTasksHeader),
   graphql`
     fragment TeamTasksHeader_team on Team {
       teamId: id
+      teamName: name
```

Comment:
woooo! declare yo needs! glad this worked for ya

Reply:
hotness